src/utils/netlify-utils.ts

```diff
@@ -0,0 +1,95 @@
+import axios from "axios"
+
+import { config } from "@config/config"
+
+import logger from "@root/logger/logger"
+
+const GITHUB_ORG_NAME = config.get("github.orgName")
+const NETLIFY_ACCESS_TOKEN = config.get("netlify.accessToken")
+
+type NetlifySiteDetails = {
+  id: string
+  password: string
+  build_settings: {
+    repo_path: string
+    repo_branch: string
```

Comment:
should we assign this to `staging` since that is the only branch we care about here?

Reply:
Hmm this is the response from netlify, which can contain branches other than those building on `staging`, so I think we'll need to maintain this info!